hatchways/team-apple-cider
# Feature public profile private lists

**What it does**

- Displays user public profile name/photo
- Profile routes/models created
- Created reusable components: Header/Body/Page
- List container now has mouse wheel horizontal scroll
- Updated API readme
- Dashboard now on `"/"` route (Issue: #69)
- Has loading spinner now instead of briefly showed protected route

**To Test:**
- Ensure db is setup & login to app
- Run `/server/scripts/generate_user_profiles.py`
- Run a GET request to `/profiles` to check profiles created
- In browser go to `/profiles/<id>` of user; name and photo should display


**What to avoid**

- Does not yet display user lists
